src/main/java/Task.java

```diff
@@ -0,0 +1,31 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718");
+    }
+
+    public void markAsDone() {
+        this.isDone = true;
+    }
+
+    public String getTask() {
+        return description;
+    }
+
+    public String toString() {
+        String str = "[";
```

Comment:
What does str mean? Maybe you can rename the variable to something more meaningful?

Reply:
Amended!